lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -25,6 +25,7 @@
 author:
 - James Tanner (@jctanner) <tanner.jc@gmail.com>
 - Loic Blot (@nerzhul) <loic.blot@unix-experience.fr>
+- Steve Jacobs (@stevejacobs) <sjacobs@brokencrew.com>
```

Comment:
Well, I learned yesterday that the author-field is to be used for people that made considerable contributions to the module.

You could add yourself to .github/BOTMETA.yml if you like to help maintain this module.